bonus_guides/F_config.md

```diff
@@ -1 +1,46 @@
-## FPO
+Since our Phoenix apps are simply Elixir applications, they  have the same configuration and structure as other Mix projects. Recall that Mix is the build tool used by most Exixir Apps.
+
+The config for our Phoenix apps is stored in a combination of .ex and .exs files.
+
+ `.ex` and  `.beam` Files
+
+Our `.ex` files are stored in the `/lib` folder of our App. These files are used to store high level configuration for our apps, such as endpint and OTP supervisor/worker config. 
+
+The `.ex` files are Elixir scripts, which are be compiled to `.beam` files that run on the Erlang Virtual Machine (BEAM, aka Bodgan's Erlang Abstract Machine). Once compiled these `.beam` files will be located in the `_build` folder of our App, the `.beam` files will be located in the `/ebin` subfolder for each Mix project in our app. For instance, since Phoenix itself is just another Elixir app, the `.beam` files for its many Mix projects are located in `/_build/dev/lib/phoenix/ebin`. Note that `/dev` would be interchanged with `/test` or `/prod` for our different environments. Also note that `phoenix` would be interchanged with the name of our app, which is also just going to be another Elixir app.
+
```

Comment:
There's a lot of great information in this paragraph and the one preceding it! I'm wondering about the placement of it, though. It seems like the main idea of this first section (greatly simplified) is that .exs behave differently from .ex files and that difference makes them ideal for configuration. (Which is great, btw.) My concern is that that message might be struggling a little to come through here.

Thoughts?